### PR TITLE
Maximization bug fixes

### PR DESCRIPTION
**Pull request by vitornesello**
Fixing bugs related to the new maximisation feature

### CODE REVIEW

File: src/decomposition.jl [hunk 1]
```diff
@@ -1,12 +1,13 @@
 set_loc_art_var(f::Formulation, constr_id::ConstrId) = setvar!(
-    f, string("local_art_", constr_id), MastArtVar; cost = 10.0,
+    f, string("local_art_", constr_id), MastArtVar;
+    cost = (getobjsense(f) == MinSense ? 1000.0 : -1000.0),
```
Comment (guimarqu) [inline src/decomposition.jl]: Did you intend to change the cost?

File: src/decomposition.jl [hunk 2]
```diff
@@ -15,11 +16,15 @@ function initialize_local_art_vars(master::Formulation,
     for (constr_id, constr) in constrs_in_form
         v = setvar!(
             master, string("local_art_of_", getname(constr)),
-            MastArtVar; cost = 10000.0, lb = 0.0, ub = Inf,
-# cost = getincval(constr), lb = 0.0, ub = Inf,
-            kind = Continuous, sense = Positive
+            MastArtVar;
+            cost = (getobjsense(master) == MinSense ? 1000.0 : -1000.0),
```
Comment (guimarqu) [inline src/decomposition.jl]: cost ?

File: src/decomposition.jl (hunk 1 above)
Comment (vitornesello) [reply src/decomposition.jl]: it was a mistake, actually function `set_loc_art_var` is not used`.

File: src/decomposition.jl (hunk 2 above)
Comment (vitornesello) [reply src/decomposition.jl]: it was a mistake.

**Pull request merged**